src/cluster/Backend.cc

```diff
@@ -23,7 +23,11 @@ using namespace zeek::cluster;
 
 
 bool detail::LocalEventHandlingStrategy::DoProcessEvent(std::string_view topic, detail::Event e) {
-    zeek::event_mgr.Enqueue(e.Handler(), std::move(e.Args()), util::detail::SOURCE_BROKER, 0, nullptr, e.Timestamp());
+    zeek::detail::EventMetadataVectorPtr meta;
+    if ( auto ts = e.Timestamp(); ts > 0.0 )
+        meta = zeek::detail::MakeEventMetadataVector(e.Timestamp());
```

Comment:
Should we as a follow-up make these events store metadata as well and remove `Timestamp`?

Reply:
Yeah, that will happen in part 2. The commit mentions "intermediary", because I wanted to keep that PR confined mostly to the non-cluster side. Resolving.